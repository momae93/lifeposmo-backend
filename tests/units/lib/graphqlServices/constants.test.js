const { constants } = require('@GraphqlService');

describe('[LIB][GRAPHQL] : constants', () => {
  it('should match snapshot lib graphql constants', async () => {
    expect(constants).toMatchSnapshot();
  });
});
