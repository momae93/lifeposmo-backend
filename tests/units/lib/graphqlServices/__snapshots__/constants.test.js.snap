// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[LIB][GRAPHQL] : constants should match snapshot lib graphql constants 1`] = `
Object {
  "SCALAR_DATE": "Date",
}
`;
